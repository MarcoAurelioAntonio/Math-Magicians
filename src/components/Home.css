.home-container {
  margin-top: 50px;
  width: 1000px;
  height: auto;
  margin-left: auto;
  margin-right: auto;
}

.welcome-1 {
  font-size: 25px;
  color: white;
  margin-bottom: 80px;
}

.welcome-2 {
  color: white;
  font-size: 25px;
}

strong {
  margin-top: 15px;
}

.home-title {
  width: 1000px;
  color: white;
  text-align: left;
  margin: 60px 0;
  font-size: 30px;
}
