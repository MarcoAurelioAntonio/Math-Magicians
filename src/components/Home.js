import React from 'react';
import './Home.css';

const Home = () => (
  <div>
    <h2>Welcome to our page!</h2>
    <p className="welcome-1">
      Lorem ipsum, dolor sit amet consectetur adipisicing elit.
      Animi cumque sit quaerat cum,ratione soluta! Eveniet, magnam
      eum ea excepturi et temporibus laborum at officia!
    </p>
    <p className="welcome-2">
      Lorem ipsum, dolor sit amet consectetur adipisicing elite.
      Animi cumque sit quaerat cum, ratione soluta! Eveniet, magnam
      eum ea excepturi et temporibus laborum at officia!
    </p>
  </div>
);

export default Home;
