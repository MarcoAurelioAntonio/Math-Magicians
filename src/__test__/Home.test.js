import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/Home';

describe('Check home function', () => {
  it('Home function renders the content', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
