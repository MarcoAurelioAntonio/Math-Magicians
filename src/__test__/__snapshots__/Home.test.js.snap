// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check home function Home function renders the content 1`] = `
<div
  className="home-container"
>
  <h2
    className="home-title"
  >
    Welcome to our page!
  </h2>
  <p
    className="welcome-1"
  >
    We will always have STEM with us. Some things will drop out of the public eye and go away, but there will always be science, engineering, and technology. And there will always, always be mathematics.
    <br />
    <br />
    <strong>
      — Katherine Johnson, African-American mathematician
    </strong>
  </p>
  <p
    className="welcome-2"
  >
    Pure mathematics is the world’s best game. It is more absorbing than chess, more of a gamble than poker, and lasts longer than Monopoly. It’s free. It can be played anywhere — Archimedes did it in a bathtub.
    <br />
    <br />
    <strong>
      —Richard J. Trudeau, mathematician
    </strong>
  </p>
</div>
`;
